---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}
<br>
<br>


Education
======
* Ph.D in Computational Biology, University of Maryland, College Park, 2024
* M.S. in Bioinformatics, Georgetown University, 2016

Research experience
======
* June 2024 - Present: Postdoctoral Associate @ Massachusetts Institute of Technology, Cambridge MA
  * Analyzing RNA-Seq data to study gene expression regulations by variants in 3' UTR
  * Investigating the feature of splicing regulatory elements in RNA sequences
  * Upgraded a white-box splicing prediction algorithm to improve the accuracy of splicing prediction in protein-coding regions
  * Evaluated the impact of alternative splicing in the splicing prediction model

* March 2022 - May 2024: Predoctoral Fellow @ National Cancer Institute, Bethesda MD
  * Analyzed nascent RNA-Seq, lariat sequencing, and PAR-CLIP data to study recursive splicing mechanism and identify potential cis- and trans-acting splicing regulators
  * Collaborated with other scientists to evaluate protein knockdown profiles through RNA-Seq analysis
  * Presented research findings at national and international conferences

* August 2019 - May 2024: PhD student/candidate @ University of Maryland - College Park, College Park MD
  * Investigated RNA sequence features to address the mixture model's ability in sequence function prediction and sequence type clustering
  * Investigated the splice site selections changes dues to consensus sequence alterations in HIV in silico using SpliceAI
  * Developed sequence feature analysis tools using mixture models, accessible on GitHub: [https://github.com/xnao25/XMUSE](https://github.com/xnao25/XMUSE)
  * Analyzed gene coexpression network in \textit{Arabidopsis} root development

* January 2016 - May 2016: Graduate research assistant @ Georgetown University, Washington DC
  * Analyzed tandem mass spectrometry data from TCGA to uncover the association between mutations and protein abundance
  * Calculated the correlations between gene expression and peptide quantification to predict mutations based solely on mass spectrometry patterns and built a web-based result visualization interface with databases


Publications
======
* Published
  * Wang, B., Mount, S.M., Latent Dirichlet allocation mixture models for nucleotide sequence analysis. NAR
Genomics and Bioinformatics, [https://doi.org/10.1093/nargab/lqae099](https://doi.org/10.1093/nargab/lqae099)

* Under revision
  * Donovan, B.T., Wang, B., Garcia, G.R., Mount, S.M., Larson, D.R. Real-time visualization of spliceosome assembly
reveals basic principles of splice site selection. bioRxiv, [https://doi.org/10.1101/2024.07.12.603320](https://doi.org/10.1101/2024.07.12.603320)
  * Garcia, G.R., Palangat, M., Moraly, J., Donovan, B.T., Wang, B., Phadke, I., Sturgill D., Chen J., Short, B., Rivero,
G.A., Swoboda, D.M., Taylor, N., McGraw, K., Larson, D.R., U2AF regulates the translation and localization of
nuclear-encoded mitochondrial mRNAs. bioRxiv, [https://doi.org/10.1101/2024.09.18.613780](https://doi.org/10.1101/2024.09.18.613780)

* In preparation
  * Wang, B., Donovan, B. T., Mount, S.M., Larson, D.R. Identification of a sequence signature for recursive splicing.
  * Gupta, K., Wang, B., Sharp, P.A., Burge, C.B., Solar-Lezama, A. Neural RNA splicing models rely on information unavailable to the splicing machinery.
  * Chang, C., Kenny, C.J., Wang, B., Burge, C.B., Global Credibility Scores Predict Alternative and High-Precision Splice
Sites in an Interpretable Model.
  
Poster Presentations
======
* Wang, B.; Wan, Y; Donovan, BT; Mount, SM; Larson, DR., (2023, Aug) Detection of a Sequence Signature for Recursive Splicing. Poster presentation at 2023 Cold Spring Harbor Eukaryotic mRNA Processing meeting.
* Wang, B.; Wan, Y; Mount, SM; Larson, DR., (2023, April) Recursive Splicing Site Sequence Feature Analysis. Poster presentation at 2023 NCI RNA Biology Symposium.
* Wang, B.; Wan, Y; Mount, SM; Larson, DR., (2023, Jan) Recursive Splicing Site Sequence Feature Analysis. Poster presentation at NCI-UMD Partnership Symposium of the Partnership for Integrative Cancer Research.
* Wang, B.; Wan, Y; Mount, SM; Larson, DR., (2022, July) Recursive Splicing Site Sequence Feature Analysis. Poster presentation at 2023 NCI RNA Biology Symposium.
* Wang, B.; Li, M.; Wang, F.; Shahan, R; Benfey, P; Mount, SM; Liu, Z. (2022, May) Root Meristem Development Analysis by Gene Coexpression Networks. Poster presentation at the COMBINE annual symposium 2022.
* Wang, B.; Mount, SM (2021, May) K-mer networks for biological sequence analysis. Poster presentation at the COMBINE annual symposium 2021.
  
Work Experience
======
* Auguest 2019 - May 2022: Teaching assistant @ University of Maryland - College Park, College Park MD
  * Delivered lectures and hands-on guidance of an entry-level undergraduate molecular and cellular biology laboratory course
  * Designed and delivered comprehensive introductory Python coding lectures for biology graduate students
  * Conducted hands-on coding sessions for practical skills improvement and provided individualized assistance on problem-solving
  * Received positive feedback from students for providing detailed coding guidance and a supportive learning environment

* September 2016 - July 2019: Bioinformatics engineer/scientist @ Burning Rock Dx, Shanghai China
  * Implemented tools to detect and interpret Next Generation Sequencing index contamination, enhancing sequencing kit quality monitoring
  * Developed a reporting system for sequencing index contamination, crucial for maintaining high-quality sequencing kit production lines
  * Implemented an algorithm for gene fusion modeling to improve the allele frequency calculation of structural variations detected from DNA-Seq
  * Created a sequencing data simulation tool to help test the accuracies of pipelines in gene mutation detection
  * Conducted DNA-Seq data analysis and statistical evaluations for sequence kit registration under the China Food and Drug Administration supervision
  * Implemented DNA-Seq analysis pipelines on Linux and improved in-house software to enhance overall pipeline performances
  
Awards and Honors
======
* UMD Dean's Fellowship (2019)
* UMD Dean's Fellowship (2020)
* NCI Graduate Student Poster Award (2023)
